### DISCUSSION TITLE
Εξέταση Σεπτεμβρίου - Τεχνολογία Λογισμικού

### DISCUSSION THREAD

**Discussion by geopala** [courses-ionio/sw#1015] [Announcements]
Καλησπέρα σας, ονομάζομαι Παλάντζας Γεώργιος με Α.Μ: Π2007051.

Επιθυμώ να εξεταστώ τον ερχόμενο Σεπτέμβρη στο μάθημά σας και ήθελα να ρωτήσω τι ισχύει με τη βαθμολόγηση.

Συγκεκριμένα με τα 20% και 10% των **Συμμετοχή στην τάξη** και **Βίντεο κουίζ**, αντίστοιχα, όπως αναφέρονται στον πίνακα.

Θα βαθμολογηθώ μόνο με τα υπόλοιπα παραδοτέα ή θα πρέπει να ακολουθήσω τη διαδικασία των quiz?(εφόσον είναι ακόμη διαθέσιμα)

Ευχαριστώ προκαταβολικά

**Comment by epidrome** [comment]
Προσπάθησε να κάνεις τα διαθέσιμα παραδοτέα όσο καλύτερα γίνεται, τα κουιζ προφανώς δεν είναι διαθέσιμα

Η εξεταστική του Σεπτ. είναι "επαναληπτική" δλδ κάποιος έχει ήδη προσπαθήσει την κανονική

**Reply by geopala** [reply]
Έχω ολοκληρώσει σχεδόν όλα τα παραδοτέα θεωρώ, εκτός των αιτημάτων στην ιστοσελίδα.

Αν είναι εύκολο, να ενημερωθούν γιατί ήδη το 1/2 είναι marked as stale.

Ευχαριστώ
